src/cli.js

```diff
@@ -3,12 +3,15 @@
 
 const sane = require('../');
 const argv = require('minimist')(process.argv.slice(2));
-const execshell = require('exec-sh');
+const spawn = require('child_process').spawn;
```

Comment:
can we use execa here, that way if this process crashes/is torn down, it will also not leak child processes.